src/Microsoft.Tye.Hosting/Microsoft.Tye.Hosting.csproj

```diff
@@ -21,6 +21,7 @@
   <ItemGroup>
     <PackageReference Include="Bedrock.Framework" Version="0.1.38-alpha.gd25d5b37ad" />
     <PackageReference Include="FeatherHttp" Version="0.1.42-alpha.gf06a8747e7" />
+    <PackageReference Include="Makaretu.Dns.Multicast" Version="0.27.0" />
```

Comment:
What's this external dependency? Are we safe to take a dependency on it?

Reply:
License is MIT: https://github.com/richardschneider/net-mdns/blob/master/LICENSE

But should go through a review before being committed.

Reply:
Yes, I wasn't sure what the policies/requirements for this org were regarding 3rd-party dependencies, hence this being more of a PoC to get feedback on the general idea and approach.  This was the only mDNS library I could find for .NET that supported the advertisement side (vs. the observing side), other than an ancient Xamarin implementation that no longer appears to be supported, and also seems dependent (on Windows) on 3rd-party components.